packages/server/dataloader/azureDevOpsLoaders.ts

```diff
@@ -507,23 +566,21 @@ export const azureDevOpsWorkItems = (
             console.log(error)
             return []
           }
-          const mappedWorkItems: AzureDevOpsWorkItem[] = returnedWorkItems.map(
-            (returnedWorkItem) => ({
-              id: returnedWorkItem.id.toString(),
-              title: returnedWorkItem.fields['System.Title'],
-              teamProject: getProjectId(new URL(returnedWorkItem.url)),
-              url: returnedWorkItem.url,
-              state: returnedWorkItem.fields['System.State'],
-              type: returnedWorkItem.fields['System.WorkItemType'],
-              descriptionHTML: returnedWorkItem.fields['System.Description']
-                ? returnedWorkItem.fields['System.Description']
-                : '',
-
-              service: 'azureDevOps'
+
+          const mappedWorkItems: AzureDevOpsWorkItem[] = await Promise.all(
+            returnedWorkItems.map(async (returnedWorkItem): Promise<AzureDevOpsWorkItem> => {
+              const mappedWorkItem = await getMappedAzureDevOpsWorkItem(
+                manager,
+                userId,
+                teamId,
+                instanceId,
+                returnedWorkItem
+              )
+              return mappedWorkItem
             })
           )
 
-          return mappedWorkItems
+          return await mappedWorkItems
```

Comment:
+1 `await` doesn't do anything here as we're already using `await` on line 570

Reply:
Removed